frontend/src/scenes/billing/v2/billingLogic.ts

```diff
@@ -168,10 +170,15 @@ export const billingLogic = kea<billingLogicType>([
 
     urlToAction(({ actions }) => ({
         '/organization/billing': (_params, _search, hash) => {
+            console.log('TEST', _search)
             if (hash.license) {
                 actions.setShowLicenseDirectInput(true)
                 actions.setActivateLicenseValues({ license: hash.license })
                 actions.submitActivateLicense()
+            } else if (_search.success) {
+                // if the activation is successful, we reload the user to get the updated billing info on the organization
+                actions.loadUser()
```

Comment:
Does this work? This runs in `urlToAction` which could be running before the billing has been loaded? At the moment we have the billing-v2 check which covers it but when we eventually remove it we may forget about this code...